semantics/executable-spec/src/Control/State/Transition/Extended.hs

```diff
@@ -15,7 +15,6 @@
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE StandaloneDeriving #-}
-{-# LANGUAGE StrictData #-}
```

Comment:
This is something I don't like about `StrictData`:

* Apparently, some data types were strict when they shouldn't have been.
* Which ones? In other words, turning off which "bangs" made the difference?